src/vision.cs

```diff
@@ -1153,6 +1258,11 @@ interface VNFaceObservation {
 		[Export ("faceObservationWithRequestRevision:boundingBox:roll:yaw:")]
 		VNFaceObservation FromBoundingBox (VNFaceObservationRequestRevision requestRevision, CGRect boundingBox, [NullAllowed] [BindAs (typeof (nfloat?))] NSNumber roll, [NullAllowed] [BindAs (typeof (nfloat?))] NSNumber yaw);
 
+		[TV (13,0), Mac (10,15), iOS (13,0)]
+		[BindAs (typeof (nfloat?))]
```

Comment:
doc suggest it's `float?` not `nfloat?`
Not a big deal since it goes thru `NSNumber` but other API are also using `float`